apps/web/src/lib/api/fetcher.ts

```diff
@@ -71,6 +71,7 @@ export function getAll<
           depth: 10, // TODO: remove this when we have a better way to handle depth for example with GraphQL
           // Needs to be bigger than 1 to get media / images
           limit: 100,
+          sort: "-createdAt",
```

Comment:
Let's have this as a parameter that is passed/sort in next.js side, I don't think sorting all queries is good.

Reply:
Makes sense, but needed I thought it needed much refactoring for the fetcher implementation that I wanted to skip it :sweat_smile: I'll do it

Reply:
Fixed